app/utils/transformer/PageTransformer.scala

```diff
@@ -56,7 +56,6 @@ trait PageTransformer {
           accTry.flatMap(_.set(capturedAnswer._1, capturedAnswer._2))
       }
 
-  // TODO - can we refactor to use an ID to get a particular DomainModelType rather than getting the entire list?
```

Comment:
I personally think this is still outstanding, but might be too large a refactor to handle this week. Am I right in thinking the transformers are primarily based on fetching entire code lists then using .find on that list against a particular value?

For larger IE013/IE015 messages this may put some serious strain on the application when storing large code lists in memory. It would be more performant to use query parameters where we can to just fetch the specific value we're after.

It may also be prudent to replicate the approach taken in manage-transit-movements-unloading-frontend for mapping a message into user answers, for consistency, understandability, etc.